packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -296,6 +300,16 @@ class InteractiveViewer extends StatefulWidget {
   /// than maxScale.
   final double minScale;
 
+  /// Changes the scrolling behavior after end of a gesture.
+  ///
+  /// Used as the coefficient of friction in the inertial translation animation.
+  /// This value was eyeballed to give a feel similar to Google Photos.
+  ///
+  /// Defaults to 0.0000135.
+  ///
+  /// Cannot be null, and must be a finite number greater than zero
+  final double scrollFriction;
```

Comment:
What about renaming to:

```suggestion
  final double scrollFrictionCoefficient;
```
The original documentation states "coefficient of friction". I believe physically this is different from just "friction". The coefficient of friction is usually represented with a value between 0 and 1.5 (https://www.engineeringtoolbox.com/friction-coefficients-d_778.html). Whereas friction is the force usually measured in Newtons.

Wdyt of this rename?
---
Resources:

From https://www.britannica.com/science/coefficient-of-friction:
> coefficient of friction, ratio of the frictional force resisting the motion of two surfaces in contact to the normal [force](https://www.britannica.com/science/force-physics) pressing the two surfaces together. It is usually symbolized by the Greek letter mu (μ). Mathematically, μ = F/N, where F is the frictional force and N is the normal force. B

It defers from just "friction" (Definitions from [Oxford Languages](https://languages.oup.com/google-dictionary-en):
> the resistance that one surface or object encounters when moving over another.

Here a comparison https://www.khanacademy.org/science/high-school-physics/two-dimensional-motion-2/friction/a/friction-ap-physics-1.